src/core/markdown/MarkdownSerializer.js

```diff
@@ -301,7 +301,7 @@ export class MarkdownSerializerState {
     // content. If `startOfLine` is true, also escape characters that
     // have special meaning only at the start of the line.
     esc(str, startOfLine) {
-        const escRegexp = this.options?.commonEscape || /[`\^+*\\\|~\[\]\{\}<>\$]/g;
+        const escRegexp = this.options?.commonEscape || /[`\^+*\\\|~\[\]\{\}<>\$_]/g;
```

Comment:
Let's add tests for this case to `markdown-editor/src/core/markdown/Markdown.test.ts`. What do you think?

Reply:
Yes, okay, I'll add tests